clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -169,6 +170,15 @@ common_args = ["-default_behaviors", "all,-bye",
                "-nostdin",
                "-recv_timeout", "20s",
                "-t", "t1"]
+if args.ipv6:
+    # To send traffic to an IPv6 system, we need to pass SIPp the system's IPv6
+    # address. There's no nice, portable way to do this, so parse ifconfig's
+    # output.
+    #
+    # We could use https://pypi.python.org/pypi/netifaces here, but that would
+    # introduce an extra dependency.
+    ipv6_addr = check_output("ifconfig eth0 | grep inet6 | grep Global | grep -Eo \"[a-f0-9]{4}:[a-f0-9:]+\" | head -n 1", shell=True).strip()
```

Comment:
This doesn't work for me on the SIPp node.  grep Global step gives

```inet6 addr: fd5f:5d21:845:1422:250:56ff:fead:bda5/64 Scope:Global``` 

but the next grep does not match anything.